### ISSUE TITLE
Accessibility: Limit Your Search Filters

### ISSUE THREAD

**Issue by VanessaVenti** [projectblacklight/spotlight#3557]
### Issue Description - WCAG standard
When expanded, the first item (e.g., creator, date) reads the name of the filter, not the count associated with the filter. Each filter should include the number of items related to the filter when reading out to screenreaders. All options under each filter either read an incorrect number or don’t read a number at all. 

WCAG Criteria: 1.1.1 Non-text Content

### Method of discovery
Siteimprove, Wave, keyboard navigation, VoiceOver etc.
VoiceOver

### Suggested solution
Ensure all items and their respective counts are read correctly

**Comment by VanessaVenti** [comment]
Ah ok, thanks @corylown! Should I close this as not planned then and add it to the Blacklight board?

**Comment by corylown** [comment]
> Ah ok, thanks [@corylown](https://github.com/corylown)! Should I close this as not planned then and add it to the Blacklight board?

I think that makes sense.
